### PR TITLE
Update archived-device-list.md

### PR DESCRIPTION
**Pull request by rekhanr**
Added HP EliteBook 830 / 840 / 850 G5 to the archived list

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/service-description/archived-device-list.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The links to devices here are for your reference only. If you want to order devi
 
 | Model  | Minimum specs  | Additional requirements  | Retirement date |
 |---------|---------|---------|---------|
+| [HP EliteBook 830 / 840 / 850 G5](https://store.hp.com/us/en/mdp/laptops/elitebook-840#!&tab=features) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR Camera or fingerprint sensor required | None | **Feb 15, 2023** |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/archived-device-list.md]:
```suggestion
| [HP EliteBook 830 / 840 / 850 G5](https://store.hp.com/us/en/mdp/laptops/elitebook-840#!&tab=features) | 128 GB / Intel i5 / 8 GB RAM | SKU with 5VS01AV, IR camera or fingerprint sensor required | None | **Feb 15, 2023** |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit

**Comment by mypil** [comment]
@jaimeo - PR has been copy edited and is ready for final review. 


Thank you.

cc: @kenwith

**Pull request merged**